EventEnabledInsurance/build/pipeline.yaml

```diff
@@ -200,6 +200,7 @@ spec:
         - name: git-source
           workspace: git-source
 
+    # push to apic
```

Comment:
No need to change anything, but I don't see the point in comments that say the same thing as the code in the line below. Personally I would only put in comments that actually explain things not already understandable from the naming

Reply:
Done